src/profile-logic/marker-schema.js

```diff
@@ -394,8 +404,118 @@ export function formatFromMarkerSchema(
       return formatPercent(value);
     default:
       console.error(
-        `A marker schema of type "${markerType}" had an unknown format "${(format: empty)}"`
+        `A marker schema of type "${markerType}" had an unknown format ${JSON.stringify(
+          format
+        )}`
       );
       return value;
   }
 }
+
+export function isDOMRenderingFormat(format: MarkerFormatType): boolean {
+  return format === 'url' || typeof format === 'object' || format === 'list';
+}
+
+function _formatDOMFromMarkerSchema(
+  markerType: string,
+  format: MarkerFormatType,
+  value: any,
+  // add elements that allow interaction (like clickable urls)
+  supportsInteraction: boolean
+) {
+  if (!isDOMRenderingFormat(format)) {
+    return <>{formatFromMarkerSchema(markerType, format, value)}</>;
```

Comment:
nit: we don't need the fragment because we're not wrapping several nodes, therefore just return the result of `formatFromMarkerSchema`